db/migrate/20150128155644_default_to_nil_for_adviser_covers_whole_of_uk.rb

```diff
@@ -1,9 +0,0 @@
-class DefaultToNilForAdviserCoversWholeOfUk < ActiveRecord::Migration
```

Comment:
Why was the migration removed? Unless it's not been deployed I would keep it present.